pkg/operator/revisionpoddeployer/unionrevisionedpod.go

```diff
@@ -0,0 +1,114 @@
+package revisionpoddeployer
+
+import (
+	"fmt"
+	"reflect"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/openshift/library-go/pkg/operator/encryption/encryptionconfig"
+	"github.com/openshift/library-go/pkg/operator/encryption/statemachine"
+)
+
+type DeployerExtended interface {
+	statemachine.Deployer
+	Disabled() bool
+}
+
+// UnionRevisionLabelPodDeployer provides unified state from multiple distinct deployers.
```

Comment:
... it's converged if all delegates are converged. Each deployer can be disabled.

What if all are disabled?

Reply:
> What if all are disabled?

we will treat it as if encryption was off